config.toml

```diff
@@ -1,21 +1,115 @@
-title        = "Tekton Pipelines"
-baseURL      = "https://tekton-dev.netlify.com"
+baseURL = "/"
+title = "Tekton"
+
+enableRobotsTXT = true
+
+# Hugo allows theme composition (and inheritance). The precedence is from left to right.
+theme = ["docsy"]
+
+# Will give values to .Lastmod etc.
+enableGitInfo = true
+
+# Language settings
+contentDir = "content/en"
+defaultContentLanguage = "en"
+defaultContentLanguageInSubdir = false
+# Useful when translating.
+enableMissingTranslationPlaceholders = true
+
 disableKinds = ["taxonomy", "taxonomyTerm"]
 
+# Highlighting config
+pygmentsCodeFences = true
+pygmentsUseClasses = false
+# Use the new Chroma Go highlighter in Hugo.
+pygmentsUseClassic = false
+#pygmentsOptions = "linenos=table"
+# See https://help.farbox.com/pygments.html
+pygmentsStyle = "tango"
+
+# Configure how URLs look like per section.
+[permalinks]
+blog = "/:section/:year/:month/:day/:slug/"
+
+## Configuration for BlackFriday markdown parser: https://github.com/russross/blackfriday
 [blackfriday]
+plainIDAnchors = true
 hrefTargetBlank = true
+angledQuotes = false
+latexDashes = true
+
+# Image processing configuration.
+[imaging]
+resampleFilter = "CatmullRom"
+quality = 75
+anchor = "smart"
+
+[services]
+[services.googleAnalytics]
+# Comment out the next line to disable GA tracking. Also disables the feature described in [params.ui.feedback].
+id = "UA-00000000-0"
+
+# Language configuration
+
+[languages]
+[languages.en]
+title = "Tekton"
+description = "Kubernetes-native CI/CD"
+languageName ="English"
+# Weight used for sorting.
+weight = 1
+
+# Everything below this are Site Params
 
 [params]
-favicon = "favicon.png"
-subtitle = "A Kubernetes-native pipeline resource"
-github = "https://github.com/tektoncd/pipeline"
-info = """
-The **Tekton Pipelines** project provides [Kubernetes](https://kubernetes.io)-style resources for declaring CI/CD-style pipelines.
-"""
-comingsoon = "This site is currently under construction. For more information on the Tekton Pipelines project in the meantime, check out the [repo on GitHub](https://github.com/tektoncd/pipeline)."
-
-[params.logos]
-hero = "horizontal-color"
-
-[params.fonts]
-sansserif = "Franklin Gothic Heavy Regular"
+copyright = "The Tekton Project"
+privacy_policy = "https://www.linuxfoundation.org/privacy/"
+
+# First one is picked as the Twitter card image if not set on page.
+# images = ["images/project-illustration.png"]
+
+# Menu title if your navbar has a versions selector to access old versions of your site.
+# This menu appears only if you have at least one [params.versions] set.
+version_menu = "Releases"
+
+# Repository configuration (URLs for in-page links to opening issues and suggesting changes)
+github_repo = "https://github.com/tektoncd/"
```

Comment:
i think we want this to be `https://github.com/tektoncd/website` it looks like it's used on pages like https://deploy-preview-14--tekton-dev.netlify.com/docs/ at the bottom to link back to here?